src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/matmul.cpp

```diff
@@ -113,11 +115,95 @@ INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MatMulFQ, MatMulFQ,
                          ::testing::Combine(
                                  ::testing::ValuesIn(input_shapes),
                                  ::testing::ValuesIn(precisions()),
+                                 ::testing::Values(MatMulType::MatMul),
                                  ::testing::Values(1), // MatMul;
                                  ::testing::Values(1), // Tokenized MatMul
                                  ::testing::Values(ov::test::utils::DEVICE_CPU)),
                          MatMul::getTestCaseName);
 
+INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MatMulEltwiseChain, MatMulEltwiseChain,
+                         ::testing::Combine(
+                             ::testing::ValuesIn(input_shapes),
+                             ::testing::ValuesIn(precisions()),
+                             ::testing::Values(MatMulType::MatMul),
+                             ::testing::Values(1), // MatMul
+                             ::testing::Values(1), // Tokenized MatMul
+                             ::testing::Values(ov::test::utils::DEVICE_CPU)),
+                         MatMul::getTestCaseName);
+
+std::vector<std::vector<ov::test::InputShape>> matmul_cascade_shapes{
+    {
+        {PartialShape{-1, -1, -1, -1}, {{2, 1, 32, 2500}, {1, 3, 80, 700}, {2, 1, 32, 2500}}},
+        {PartialShape{-1, -1, -1, -1}, {{1, 2, 2500, 128}, {1, 3, 700, 150}, {1, 2, 2500, 128}}},
+        {PartialShape{-1, -1, -1, -1}, {{1, 1, 128, 64}, {1, 3, 150, 128}, {1, 1, 128, 64}}},
+    },
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MatMulEltwiseChainCascade, MatMulEltwiseChainCascade,
+                         ::testing::Combine(
+                             ::testing::ValuesIn(matmul_cascade_shapes),
+                             ::testing::ValuesIn(precisions()),
+                             ::testing::Values(MatMulType::MatMul),
+                             ::testing::Values(1), // MatMul
+                             ::testing::Values(1), // Tokenized MatMul
+                             ::testing::Values(ov::test::utils::DEVICE_CPU)),
+                         MatMul::getTestCaseName);
+
+std::vector<std::vector<ov::test::InputShape>> fc_input_shapes{
+    {
+        {PartialShape{-1, -1, -1, 2500}, {{2, 1, 32, 2500}, {1, 3, 80, 2500}}},
+        {{}, {{2500, 256}}}
+    },
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_Snippets_FullyConnected, MatMul,
```

Comment:
What do you think about the separate file for `FC` test instances? Does it make sense?
I suggest this splitting because when I was working on dynamic Brgemm support with BF16/INT8, it was difficult to figure out the tests because of their number

Reply:
When I reviewed your changes in this file, I also thought about separation :) I agree, let's create a separate file for FC related instances: this will not only improve readability, but also reduce merge conflicts of our changes.
Thanks for the idea!

Reply:
> reduce merge conflicts of our changes

Exactly! Thanks a lot! ❤️